health/notifications/hangouts/README.md

```diff
@@ -0,0 +1,33 @@
+# Hangouts Chat
+
+This is what you will get:
+![Netdata on Hangouts](https://imgur.com/a/FVlZ3F0)
```

Comment:
This image wasn't loading here on GitHub, and we prefer images be on their CDN anyway, so here's the new Markdown.

```suggestion
![Netdata on Hangouts](https://user-images.githubusercontent.com/1153921/66427166-47de6900-e9c8-11e9-8322-b4b03f084dc1.png)
```